lizadams-cary.md

```diff
@@ -0,0 +1,10 @@
+# Cary, NC
```

Comment:
It is beautiful - 80 degrees!  @WilliamBonner - How is the weather in your hometown?